.github/workflows/ci-code.yml

```diff
@@ -133,3 +133,55 @@ jobs:
         verdi devel check-load-time
         verdi devel check-undesired-imports
         .github/workflows/verdi.sh
+
+
+  test-pytest-fixtures:
+    # Who watches the watchmen?
```

Comment:
We also have a `tests/test_conftest.py` :)

Reply:
Haha, yeah indeed, that one is hilarious 😂